tests/lib/tools/fs-state

```diff
@@ -72,3 +90,25 @@ clean_file() {
         mv "$file.back" "$file"
     fi
 }
+
+if [ $# -eq 0 ]; then
+	show_help
+	exit 1
+fi
+
+action=
+while [ $# -gt 0 ]; do
+	case "$1" in
+		-h|--help)
```

Comment:
Could you tweak this to include the empty string `''` so that it can be invoked without arguments please?

Reply:
done